src/main/java/ml/shifu/shifu/core/processor/TrainModelProcessor.java

```diff
@@ -1527,6 +1532,11 @@ public TailThread(String[] progressLogs) {
                     LOG.error("Error in delete progressLog", e);
                 }
             }
+            bw = new BufferedWriter(new FileWriter(filePath, false));
+            bw.write("Trainer,Epoch,Training_Error,Validation_Error");
+            bw.newLine();
+            bw.flush();
+            bw.close();
```

Comment:
could you wrapper such close method in finally. Shifu supports JDK 1.6 for a broad use case, no try close in JDK 1.8